src/store/data.js

```diff
@@ -164,11 +193,13 @@ export default {
 				return false
 			}
 
-			const row = res.data
-			const rows = state.rows
-			const index = rows.findIndex(r => r.id === row.id)
-			rows[index] = row
-			commit('setRows', [...rows])
+			if (tableId) {
+				const row = res.data
+				const rows = state.rows[tableId]
+				const index = rows.findIndex(r => r.id === row.id)
+				rows[index] = row
+				commit('setRows', {tableId, rows:[...rows]})
+			}
 			return true
 		},
 		async insertNewRow({ state, commit, dispatch }, { viewId, tableId, data }) {
```

Comment:
Need to figure out how to handle Views. Using the tableId as a key for state.columns and state.rows doesn't really work

Reply:
We can't just use the View id as a key, since it might collide with a Table id